python/paddle/tensor/__init__.py

```diff
@@ -151,6 +151,7 @@
 from .math import square  # noqa: F401
 from .math import stanh  # noqa: F401
 from .math import sum  # noqa: F401
+from .math import nansum  # noqa: F401
```

Comment:
for paddle.Tensor.nansum, we should add nansum in list of tensor_method_func below